PyInstaller/compat.py

```diff
@@ -74,6 +75,9 @@
                      'libpython%d.%dm.so.1' % _pyver,
                      'libpython%d.%d.so.1.0' % _pyver,
                      'libpython%d.%dm.so.1.0' % _pyver}
+elif is_hpux:
+    PYDYLIB_NAMES = {'libpython%d.%d.a' % _pyver,
```

Comment:
I assume the `.a` is wrong and should be removed.

`.a` is a static library on most Unix systems. Is this different at HP-UX?